open-company/open-company-web
# List users roster

Review with:
- [Auth](https://github.com/open-company/open-company-auth/pull/85)

From [QA doc](https://paper.dropbox.com/doc/Post-beta-bugs-and-comments--AeU_GqSGUBWtqpEBVWd04VhzAg-SPQWk76sbpqMUySRddhEm)
item:
> Show contributors and viewers the team list/roles
Also show invite panel to contributors (not viewers)

To test:
- signup as admin user A
- with user A load the org
- open the menu
- [x] do you get all 4 items: admin settings, invite people, team management and integrations?
- open org settings
- [x] does it work? Good
- open invite
- [x] invite a contributor: user B
- [x] invite a viewer: user C
- go to Team management
- [x] do you see all the users?
- [x] can you change their roles?
- put back the roles: user B contributor, user C viewer
- open integrations
- [x] does it work?
- now onboard user B
- open the menu
- [x] do you see these 2 items only: view team and invite people?
- open View team
- [x] title is View team?
- [x] do you see all the users with the right roles?
- [x] can you NOT change the user roles?
- click the invite button at the top
- [x] can you invite? Good
- onboard user C
- open the menu
- [x] do you see only View team?
- open it
- [x] title is View team? Good
- [x] no Invite button at the top?
- [x] can you see all the users with the right roles?
- [x] can you NOT change the user roles?
